src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,23 @@
+package duke.task;
+
+public class Event extends Task {
+    protected String from;
+    protected String to;
+
+    public Event(String description, String from, String to, boolean isComplete) {
+        super(description);
+        this.from = from;
+        this.to = to;
+        this.isComplete = isComplete;
+    }
+
+    @Override
+    public String toFileFormat() {
+        return String.format("E | %d | %s | %s to %s", isComplete ? 1 : 0, description, from, to);
+    }
+
+    @Override
+    public String toString() {
+        return "E | " + (isComplete ? 1 : 0) + " | " + description + " | " + from + " to " + to;
```

Comment:
Perhaps could make use of polymorphism here so that `isComplete` and `description` can be made `private` rather than `protected` for better encapsulation, i.e. call `super.toString()` here. Same for `toFileFormat()`.

Reply:
I think since the task are similar in terms of their file format, as said by YiBai , I think you could have Task handle the similarities and then use inheritance to add the details specific to the various task types

```suggestion
        public class Task {
           /**
              some implementation of filefomrat
           */
        }
        
        public class Event extends Task {
            /**
                call parent file format with super.fileformat()
            */
        }
```